.github/workflows/core.yml

```diff
@@ -55,7 +55,6 @@ jobs:
     if: (github.repository == 'Firebase/firebase-ios-sdk' && github.event_name == 'schedule') || github.event_name == 'pull_request'
 
     runs-on: macos-11
-    # TODO(v9): Remove `env` after publishing `HeartbeatLoggingTestUtils`.
     env:
       POD_LIB_LINT_ONLY: 1
```

Comment:
Nit: same comment here

Reply:
oof, sorry I missed these! 🤦

Reply:
Done in [2578961](https://github.com/firebase/firebase-ios-sdk/pull/9554/commits/2578961b587e5620ab16688aea7ce3906831a3c3)